Yoast/wordpress-seo
# FRO-11 update styling form fields settings pages

## Context


* Update the input fields, text areas, dropdowns,  buttons, title separator and toggle switches in the settings pages to the new styling

## Summary


This PR can be summarized in the following changelog entry:

* Update the components in the settings pages to the new styling

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Pull the `14291-update-styling-form-fields` in from the javascript repo
* Pull this repo
* Link the javascript repo
* Build this repo
* Check the input fields, text areas, dropdowns,  buttons, title separator and toggle switches are in line with the design: https://sketch.cloud/s/KLz2A/a/W0Zj00/play


## UI changes

## Documentation

## Quality assurance


Fixes #14291
